gef.py

```diff
@@ -8784,6 +8823,102 @@ def do_invoke(self, argv):
 
         return
 
+
+@register_command
+class HighlightCommand(GenericCommand):
+    """
+    This command highlights user defined text matches which modifies GEF output universally.
+    """
+    _cmdline_ = "highlight"
+    _syntax_ = "{} (add|del|list|clear|regex)".format(_cmdline_)
+    _aliases_ = ["hl"]
+
+    def __init__(self):
+        super(HighlightCommand, self).__init__(prefix=True)
+
+    def do_invoke(self, argv):
+        self.usage()
+
+
+@register_command
+class HighlightListCommand(GenericCommand):
+    """Show the current highlight table with matches to colors."""
+    _cmdline_ = "highlight list"
+    _aliases_ = ["highlight ls", "hll"]
+    _syntax_ = _cmdline_
+
+    def do_invoke(self, argv):
+        print_highlight_table()
+
+
+@register_command
+class HighlightClearCommand(GenericCommand):
+    """Clears the highlight table, removing all matches."""
+    _cmdline_ = "highlight clear"
+    _syntax_ = _cmdline_
+
+    def do_invoke(self, argv):
+        highlight_table.clear()
+
+
+@register_command
+class HighlightAddCommand(GenericCommand):
+    """Adds a match to the highlight table."""
+    _cmdline_ = "highlight add"
+    _syntax_ = "{} [match] [color]".format(_cmdline_)
+    _aliases_ = ['highlight set']
+    _example_ = "{} 41414141 yellow".format(_cmdline_)
+
+    def do_invoke(self, argv):
+        if len(argv) < 2:
+            return self.usage()
+
+        match, color = argv
+        highlight_table[match] = color
+
+
+@register_command
+class HighlightRemoveCommand(GenericCommand):
+    """Removes a match in the highlight table."""
+    _cmdline_ = "highlight remove"
+    _syntax_ = "{} [match]".format(_cmdline_)
+    _aliases_ = [
+        'highlight delete',
+        'highlight del',
+        'highlight unset',
+        'highlight rm'
+    ]
+    _example_ = "{} remove 41414141".format(_cmdline_)
+
+    def do_invoke(self, argv):
+        if not argv:
+            return self.usage()
+
+        highlight_table.pop(argv[0], None)
+
+
+@register_command
+class HighlightRegexCommand(GenericCommand):
+    """Toggles RegEx support for highlights."""
+    _cmdline_ = "highlight regex"
+    _syntax_ = "{} [on/off/status]".format(_cmdline_)
```

Comment:
Not optional

Reply:
Use `|`. Try to make this look like a man page.

Reply:
👍